modules/k8s-nmstate-troubleshooting-dns-disconnected-env.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-troubleshooting-node-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting-dns-disconnected-env_{context}"]
+= Troubleshooting DNS connectivity issues in a disconnected environment
+
+If you experience DNS connectivity issues when configuring `nmstate` in a disconnected environment, you can configure the DNS server to resolve the list of name servers for the domain `root-servers.net`.
+
+== Configuring the bind9 DNS named server  
+
+For a cluster configured to query a `bind9` DNS server, you can add the `root-servers.net` zone to a configuration file that contains at least one NS record.
+For example you can use the `/var/named/named.localhost` as a zone file that already matches this criteria.
+
+.Procedure
+
+. Add the `root-servers.net` zone at the end of the `/etc/named.conf` configuration file by running the following command:
++
+[source,terminal]
+----
+$ cat >> /etc/named.conf <<EOF
+zone "root-servers.net" IN {
+    	type master;
+    	file "named.localhost";
+};
+EOF
+----
+
+. Restart the `named` service by running the following command:
++
+[source,terminal]
+----
+$ systemctl restart named
+----
+
+. Confirm that the `root-servers.net` zone is present by running the following command:
++
+[source,terminal]
+----
+$ journalctl -u named|grep root-servers.net
+----
++
+.Example output
+[source,terminal]
+----
+Jul 03 15:16:26 rhel-8-10 bash[xxxx]: zone root-servers.net/IN: loaded serial 0
+Jul 03 15:16:26 rhel-8-10 named[xxxx]: zone root-servers.net/IN: loaded serial 0
+----
+
+. Verify that the DNS server can resolve the NS record for the `root-servers.net` domain by running the following command:
++
+[source,terminal]
+----
+$ host -t NS root-servers.net. 127.0.0.1
+----
++
+.Example output
++
+[source,terminal]
+----
+Using domain server:
+Name: 127.0.0.1
+Address: 127.0.0.1#53
+Aliases:
+root-servers.net name server root-servers.net.
+----
+
+== Configuring the dnsmasq DNS server
```

Comment:
This is seems like a different procedure and needs to go into a different module.